helm/kube-image-keeper/templates/registry-deployment.yaml

```diff
@@ -48,8 +48,16 @@ spec:
                   key: secret
             - name: REGISTRY_STORAGE_DELETE_ENABLED
               value: "true"
+            {{- if (not (empty .Values.registry.persistence.s3))}}
             - name: REGISTRY_STORAGE
               value: s3
+            {{- end}}
+            {{- if (not (empty .Values.registry.persistence.gcs))}}
+            - name: REGISTRY_STORAGE
+              value: gcs
+            - name: REGISTRY_STORAGE_GCS_KEYFILE
```

Comment:
I don't think its a real issue. The registry will output an error anyway if the secret is missing. And otherwise it will work but silently using tmpfs, which could be unexpected from a user perspective.